tests/Twig/GlobalVariablesTest.php

```diff
@@ -109,4 +109,21 @@ public function testInvalidArgumentException()
 
         new GlobalVariables('foo');
     }
+
+    /**
+     * @dataProvider getMosaicBackgroundProvider
+     */
+    public function testGetMosaicBackground($expected, $image)
+    {
+        $globalVariables = new GlobalVariables($this->pool, $image);
+        $this->assertEquals($expected, $globalVariables->getMosaicBackground());
+    }
+
+    public function getMosaicBackgroundProvider()
+    {
+        return [
+            'background is null' => [null, null],
+            'background is null' => ['image.png', 'image.png']
```

Comment:
This does not look correct

Reply:
Right, I commited too fast, it was corrected in next commit.